source/JointMilitarySymbologyLibraryCS/jmsml.config

```diff
@@ -161,5 +161,409 @@
       <AmplifierValue ID="REDUCED" Label="-">1</AmplifierValue>
       <AmplifierValue ID="REINFORCED_REDUCED" Label="+/-">2</AmplifierValue>
     </AmplifierValues>
+
+    <SchemaTypes>
```

Comment:
Just a general observation - and this is probably a good time to remind everyone on the team about these PR reviews/practices:

1. please be nice to your reviewer who invested time&energy in reviewing your work product 
2. please do not to reply with long/complex comments - that require further time of the reviewer - since many of  the review comments are just observations/suggestions from someone who is not intimately familiar with the work product like you are
3. Not every review comment needs a reply (we do however recommend a `"Fixed in commit #"` if you did fix)
4. Try to limit debate/discussion to those items where a review item must be completed before being merged (and there is a disagreement)
5. If you don't want to do a review item as part of the PR, create an issue for tracking and/or discussion (and link to the PR comment so we can trace back)

Not picking on you individually here, these are things we all need to improve on, so I just wanted to mention/remind as part of this discussion (& I'll update our Github review procedures now to make sure this is all captured there).